modules/navier_stokes/src/actions/NSFVAction.C

```diff
@@ -489,6 +489,7 @@ NSFVAction::NSFVAction(InputParameters parameters)
     _turbulence_handling(getParam<MooseEnum>("turbulence_handling")),
     _porous_medium_treatment(getParam<bool>("porous_medium_treatment")),
     _porosity_name(getParam<MooseFunctorName>("porosity")),
+    _smooth_porosity_name(isParamValid("smoothing_layers") ? NS::smooth_eps : _porosity_name),
```

Comment:
I would name this data member something different. Maybe `_porosity_functor_name`?